# scout_plane msg 

# 无人机编号
int8[] UAV_NUM

# GPS位置信息
sensor_msgs/NavSatFix target_GPS

# 数字面值
int8 num_val




